src/mon/Monitor.cc

```diff
@@ -1887,11 +1896,7 @@ void Monitor::win_election(epoch_t epoch, set<int>& active, uint64_t features,
   // when monitors are call elections or participating in a paxos
   // round without agreeing on who the participants are.
   monmon()->election_finished();
-  for (vector<PaxosService*>::iterator p = paxos_service.begin();
-       p != paxos_service.end(); ++p) {
-    if (*p != monmon())
-      (*p)->election_finished();
-  }
+  _finish_svc_election();
```

Comment:
so `monmon()->election_finished()` would be called twice? it will be fine i guess. as the second call will be basically a no-op.